src/io_selection.xml

```diff
@@ -0,0 +1,4 @@
+<lcgdict>
+  <selection>
+  </selection>
```

Comment:
I suppose this is here mainly to make the whole CMake functions work? I think this should be populated with the `Reader` and `Writer` classes, which should (hopefully) be possible with the changes suggested in the other comments.

Reply:
Fixed